Makefile

```diff
@@ -73,15 +72,19 @@ build_docs_container:
 
 .PHONY: gen_adoc_tar
 gen_adoc_tar: synopsis_docs build_docs_container
-	cd docs && docker run -e MINISHIFT_VERSION=$(MINISHIFT_VERSION) -tiv $(shell pwd)/docs:/docs:Z minishift/docs clean adoc_tar
+	cd docs && docker run -e MINISHIFT_VERSION=$(MINISHIFT_VERSION) -tiv $(shell pwd)/docs:/home/docs:Z minishift/docs clean adoc_tar
 
 .PHONY: gen_docs
 gen_docs: synopsis_docs build_docs_container
-	cd docs && docker run -e MINISHIFT_VERSION=$(MINISHIFT_VERSION) -tiv $(shell pwd)/docs:/docs:Z minishift/docs gen
+	cd docs && docker run -e MINISHIFT_VERSION=$(MINISHIFT_VERSION) -tiv $(shell pwd)/docs:/home/docs:Z minishift/docs gen
```

Comment:
that's a volume mount. It mounts your local checkout into the container in order to build the docs in there. And yes, _gen_ is the Rakefile target.